docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+
+- Remote clients can then access the custom channel using an 'http://' URL instead of a 't3://' URL.
+
+- Review the [Security Notes](#security-notes) below.
+
+#### Kubernetes Node Ports
+
+- Configure a custom channel for the T3 protocol in WebLogic that specifies an external address and port that are suitable for remote client use.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+- Deploy a Kubernetes NodePort to publicly expose the WebLogic ports. See [Setting up a NodePort](#setting-up-a-nodeport).
+
+- Review the [Security Notes](#security-notes) below.
+
+### Adding a WebLogic Custom Channel
+
+#### When is a WebLogic Custom Channel needed?
+
+WebLogic implicitly creates a multi-protocol default network channel that spans the Listen Address and Port fields specified on each server in the cluster. Here are some reasons you may need to configure an additional dedicated WebLogic custom channel to handle remote EJB or JMS client network traffic:
+
+- A default channel does not provide a way to configure an external address and port, but external listen address and/or port configuration is needed when a channel's configured listen address and/or port would not work if used to form a URL in the remote client.
+  - This is because remote EJB and JMS clients internally use their client's channel's configured network information to reconnect to WebLogic when needed. (The EJB and JMS clients do not always use the initial URL specified in the client's JNDI context.)
+
+- A default channel exposes multiple protocols but it's a best practice to limit the protocols that can be accessed by external clients.
+
+- A default channel may often be left insecure if it is only accessible internally, but it is often desirable to secure externally accessible channels (two-way SSL).
+
+#### Configuring a WebLogic Custom Channel
+
+The basic requirements for configuring a custom channel for remote EJB and JMS access are:
+
+- Configure a T3 protocol network-access-point (NAP) with the same name and same port on each server (the operator will set the listen address for you).
+
+- Configure the external listen address and port on each NAP to match the address and port component of a URLs your clients can use. For example, if you are providing access to remote clients via a load balancer then these should match the address and port of the load balancer.
+
+- If you want WebLogic T3 clients to tunnel through HTTP, then enable HTTP tunneling on each NAP. This is often necessary for load balancers.
+
+- Do _NOT_ set `outbound-enabled` to `true` on the network access point (the default is `false`).
+
+- Ensure you haven't enabled `calculated-listen-ports` for WebLogic dynamic cluster servers.
+
+For example, here is a snippet of config.xml for channel 'MyChannel' defined for a WebLogic dynamic cluster named 'cluster-1':
```

Comment:
put filenames and other technical names in backticks

Reply:
done